OpenRA.Mods.Common/Commands/PlayerCommands.cs

```diff
@@ -35,19 +35,32 @@ public void WorldLoaded(World w, WorldRenderer wr)
 
 		public void InvokeCommand(string name, string arg)
 		{
+			if (world.LocalPlayer == null)
+				return;
+
+			if (world.LocalPlayer.WinState != WinState.Undefined)
+			{
+				Game.Debug("This command ({0}) is disabled.".F(name));
+				return;
+			}
+
 			switch (name)
 			{
 				case "pause":
-					if (Game.IsHost || (world.LocalPlayer != null && world.LocalPlayer.WinState != WinState.Lost))
-						world.IssueOrder(new Order("PauseGame", null, false)
-						{
-							TargetString = world.Paused ? "UnPause" : "Pause"
-						});
+					if (!Game.IsHost)
```

Comment:
This is inconsistent with 

https://github.com/OpenRA/OpenRA/blob/7ebea32be9989048abf999231be7c9d8560d3eee/OpenRA.Mods.Common/Widgets/Logic/Ingame/Hotkeys/PauseHotkeyLogic.cs#L33-L35

Reply:
@LipkeGu You haven't adressed this yet?

Reply:
From IRC:
> [18:09] <+pchote> the pr as it is now is quite wrong
[18:09] <+pchote> it will only work for non-specator hosts who haven't won or lost

Reply:
https://github.com/OpenRA/OpenRA/pull/14657/commits/e30b0b34ba3c5a58f2df755363632979920eebd9#diff-64dfb1186dd16c8b0c9c86e777d2b2ca

Reply:
What is now the problem with this pr yes i adressed the notice... as linked about.

Reply:
The logic as it is currently written here will disable the pause command for everyone except the game host, and for the game host too if they are a spectator.  That isn't what we want here.

Reply:
so i should only change the game state.

Reply:
I assume both line must be the same